### DISCUSSION TITLE
Dataview implicit fields don't seem to be working inside the new dataview query

### DISCUSSION THREAD

**Discussion by isle9** [RafaelGB/obsidian-db-folder#187] [General]
**Describe the bug**
Dataview implicit fields don't seem to be working inside the new dataview query.

**To Reproduce**
`FROM !"_system" WHERE type = this.file.link` doesn't work in db-folder, but it does in a normal dataview query.

**Desktop (please complete the following information):**
 - OS: [Win 11]
 - Version [1.8.0]

**Comment by RafaelGB** [comment]
"this" implies the context of the current file, but the query does not support it yet, maybe with future dataview versions. The context could be the own database file
